media_kit/lib/src/player/native/core/execmem_restriction.dart

```diff
@@ -0,0 +1,45 @@
+/// This file is a part of media_kit (https://github.com/media-kit/media-kit).
+///
+/// Copyright © 2021 & onwards, Hitesh Kumar Saini <saini123hitesh@gmail.com>.
```

Comment:
You can have your own name in the license header. This is bit of something that we do:

- https://github.com/media-kit/media-kit/blob/21d2853fd3bfce6a67249850d8ff2aee43d00730/media_kit/lib/src/player/web/utils/hls.dart#L3
- https://github.com/media-kit/media-kit/blob/21d2853fd3bfce6a67249850d8ff2aee43d00730/video_player_media_kit/lib/video_player_media_kit.dart#L3

Reply:
Ah, sorry, I just copied the header and didn't think of that. I'll change it.